### ISSUE TITLE
Alt text character limit error not clearly shown

### ISSUE THREAD

**Issue by lapineige** [pixelfed/pixelfed#2490]
Version : 0.10.9

Alternative text is currently limited to 140 characters: https://github.com/pixelfed/pixelfed/issues/2407
When exceeding that limit, it is possible to save the alt text, no error message is shown. But if try to post the picture, there is an error message: `Oops something went wrong ! The given data was invalid.`

Problem : This message is unclear, not helpful to fix the issue, and users should not be able to save a too long alt text (and the limit should be shown in a better way, it's too currently).
